### ISSUE TITLE
string identifier for company ids

### ISSUE THREAD

**Issue by dlech** [NordicSemiconductor/bluetooth-numbers-database#84]
It would be useful if companies had an `"identifier"` property like services and characteristics. This would allow programmatic lookup of CIDs by name. The name property doesn't seem useful for this as it could change over time.

**Comment by dinesharjani** [comment]
Yes, you're not wrong. But when we got started with this project, our goal was to get it to be as 'close as possible' to the Bluetooth spec - our goal was for this project to be adopted by the community, which I have to say it broadly has, at least by my expectations.

If you have a specific idea of how to extend the API, perhaps witha /v2 option and so on, let us know.

**Comment by dinesharjani** [comment]
Yes, but how would that work? What would the guidelines be? What would the values be for the current attributes covered? How would we fill the identifier for instances covered by Bluetooth SIG? I meant it in a more 'formal' sense.

**Comment by reynoldslaw** [comment]
Apple airtags show initially with no name on nrf connect 2.6.7 build 34 for ios. Once connected the name at the bottom updates as FindMy.  Yet the name in the list cannot be filtered.

Another issue is in the RSSI tab, the colors on the left do not match the colors of the RSSI lines.

When selecting a single device, all RSSI lines turn grey, and the selected device also remains gray when it should be the only one in color.

**Comment by dinesharjani** [comment]
@reynoldslaw Apple has multiple Reserved Service(s). And since they're not documented / published in any one place (to my knowledge), they could potentially be changed, even though I agree that's not a likely thing. But I can look into it.

**Comment by AimSD23nSiR** [comment]
Ghost

**Comment by BoesesB** [comment]
Derjenige der Uncle Höehen Dubledam inszeniert gehört ebenfalls erschossen ✌️😎 ich bin nicht blöd....BKA ermittelt schon ....!

**Comment by AimSD23nSiR** [comment]
Is abuse and stalking what you do against me. I have already contacted Git Hub Support with my ID card and Nordic Semic Conductor... you abuse me for IT forensics and other purposes.

Lawyer is involved.

Sebastian

**Comment by BoesesB** [comment]
Ich bin Lebensgefährlich fragmatisch legiert mit irgendwelchen Menschen per nRF und haltet mich fest im IT Modular...

Ich hab mich auch mit Ausweis ID hier an dem Support gewendet....und werde ohne grund gestalket....es ist nix gegen ihnen persönlich.... nur da muss was unternommen werden.👍
